### PR TITLE
Update libraries.md

### PR DESCRIPTION
**Pull request by fjy**
Adding embedded-druid information in helper libraries

### CODE REVIEW

**Comment by drcrallen** [comment]
:+1:

File: docs/content/development/libraries.md [hunk 1]
```diff
@@ -64,7 +64,6 @@ Community Helper Libraries
 
 * [madvertise/druid-dumbo](https://github.com/madvertise/druid-dumbo) - Scripts to help generate batch configs for the ingestion of data into Druid
 * [housejester/druid-test-harness](https://github.com/housejester/druid-test-harness) - A set of scripts to simplify standing up some servers and seeing how things work
-* [mingfang/docker-druid](https://github.com/mingfang/docker-druid) - A Dockerfile to run the entire Druid cluster
```
Comment (drcrallen) [inline docs/content/development/libraries.md]: why is this being removed?

Comment (fjy) [inline docs/content/development/libraries.md]: i dont believe this is being maintained

Comment (fjy) [inline docs/content/development/libraries.md]: it hasn't seen any updates in 2+ years

**Comment by drcrallen** [comment]
:+1:

**Pull request merged**